### DISCUSSION TITLE
usage of cloud storage

### DISCUSSION THREAD

**Discussion by chrysenteron** [opengisch/QField#3862] [Q&A]
Hi, since my project itself is just between 800 and 900 MB on my computer, I thought 1 GB of cloud Storage should be enough (considering switching to paid plan). But when I open the project in the internet browser it says "You are using 0.08% (7,608.76MB/100,000MB) of your total storage quota." 
Now, I'm not sure how these numbers make sense. Is that supposed to mean 7.6 GB out of 100 GB? Then how does it get to that much storage? Does the file versioning count too? But with currently 1 to 3 file versions that seems like a lot, since it's almost 10x of what's on my computer. Also, how is this 0.08 % of 100 GB? Should that maybe be 8%? 
I need to make an estimate of what the paid plan will cost for my organization, but as it stands, I find it very difficult to predict the cost of file hosting. Is there a way to get a good estimate how much storage will actually be needed?

**Comment by suricactus** [comment]
Hey @chrysenteron. Sorry for bringing up confusion.

In short you are right, the versions are also taken into account. Note usually there is also a copy of your project which is the package for QField, which contains all files except attachments.

Since the last week incident we temporarily disabled deleting older versions, so as a result we are storing a bit more versions than usual. Therefore the reported storage is higher than usual. By the end of next week the reported storage will be as expected.
